README.md

```diff
@@ -15,3 +15,11 @@ See the [Maven Project Documentation](http://developer.jasig.org/projects/jspc-m
 ## JSP Compilers
 
 The available JSP compilers can be found by browsing [org.jasig.mojo.jspc GroupId](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.jasig.mojo.jspc%22).
+
+### Issues
+
+The plugin is limited to supporting JSP syntax that is supported by the corresponding compiler, which you should make sure also matches the compiler found on your servlet container installation. For any issues refer to the issue tracker for the corresponding compiler.
```

Comment:
I know that personally I was unsure as to what extent the maven plugin actually managed things. Now that I was forced to dig through it I have a much better grap of things. I felt like this blurb helped clarify that its the compilers themselves that are responsible for JSP syntax issues.  For an inexperienced user there still are two problems though. Finding the actual compiler version being used and finding their issue tracker.  I'm not sure how feasible it is to link to issue trackers that might move but maybe it would be useful to explicitly document the compilers versions in the readme? What do you think.